python/ray/node.py

```diff
@@ -93,6 +93,10 @@ def __init__(self,
                 "IP address when connecting to an existing raylet; i.e., when "
                 "head=False and connect_only=True.")
 
+        if ray_params._lru_evict:
+            assert (connect_only or
+                    head), "LRU Evict can only be passed into the head node."
```

Comment:
Why should it only be on the head node? I believe the worker nodes have to specify their own internal configs, and they should all match the head node's.

Reply:
Oh, #8921 moves the internal config into GCS--so only the head node will be able to specify this.

Reply:
Oh gotcha, should we wait to merge this one then?